content/docs/quickstart/console.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: Signing Up
+enableTableOfContents: true
 ---
 
-In order to launch Neon projects you need to sign up on <https://console.neon.tech> and
-click on the New Project button.
+In order to launch Neon projects, you need to sign up on <https://console.neon.tech> and click on the New Project button.
 
-Neon Project will automatically spin up compute and you will be able to query it with the SQL Editor, psql, or from your application.
+A Neon Project will automatically spin up compute, and you will be able to query it with the SQL Editor, psql, or from your application.
 
-All the operations that are available to you in the web UI are also available via API. You will need to generate a toke in the Settings of the console.
+All the operations that are available to you in the web UI are also available via API. You will need to generate a token in the Settings of the console.
```

Comment:
@LizardWizzard 

Some of the best practices are implemented in our internal technical writing team, this would include us having weekly meet-ups to choose an owner of documentation that will dedicate some of their time to the docs that week. Other strategies include having a date published to know when the last time your documentation was updated or having a Kanban board to know what documentation is needed and a priority.
----

I have included the changes to the last sentence, although we would like to discuss adding new content to our next meeting as currently, we are doing documentation cleanup (referring to Digital Ocean Styling Guide), and site architecture organization (ex. which topics go under what heading).

Please refer to this commit, https://github.com/neondatabase/website/pull/59/commits/becd5711ee3813805c39fd9698e36d60c31b7c23.